core-bundle/contao/library/Contao/Image.php

```diff
@@ -93,7 +93,7 @@ public static function getPath($src)
 			$filename = pathinfo($src, PATHINFO_FILENAME);
 
 			// Prefer SVG icons
-			if (file_exists($projectDir . '/assets/contao/images/' . $filename . '.svg'))
+			if (file_exists($webDir . '/assets/contao/images/' . $filename . '.svg'))
```

Comment:
This is actually a Contao 2 BC layer. Removing it would save us a lot of `file_exists()` calls.